.github/actions/spelling/excludes.txt

```diff
@@ -0,0 +1,84 @@
+# See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-excludes
```

Comment:
Same with this file? Why not having some sensible/comprehensive presets, instead of having to provide this on a per-repo basis?

Reply:
As with the other file, while it's true that most of the items in here came from https://github.com/check-spelling/spell-check-this/tree/prerelease/.github/actions/spelling, some of them didn't. For this repository, I believe that was (suggested in https://github.com/check-spelling/ceph-nvmeof/actions/runs/5810793649#summary-15752525327):
https://github.com/ceph/ceph-nvmeof/blob/f235a3b9805ef737030034a2f607e6c86f10f202/.github/actions/spelling/excludes.txt#L5

Pretty much any rule I've had has cases where it's wrong. I'm definitely still thinking about the tradeoffs, but at the moment, I think the complexity of having configuration hidden in the app is something I don't want. As it happens, my biggest regret about check-spelling is that through 0.0.21 (the version here) there's a hard-coded pattern for `\\[nrt]` which is right about as often as it is wrong -- in 0.0.22, I'm removing that from the core and moving it to: https://github.com/ceph/ceph-nvmeof/blob/f235a3b9805ef737030034a2f607e6c86f10f202/.github/actions/spelling/candidate.patterns#L572-L574

These are all tradeoffs. There's a default dictionary which I *don't* force people to maintain repository to repository (although I do allow them to replace it if they're so-inclined), but it has all sorts of archaic entries which means I had to add code to allow users to remove items from the dictionary: https://github.com/ceph/ceph-nvmeof/blob/f235a3b9805ef737030034a2f607e6c86f10f202/.github/actions/spelling/reject.txt -- in theory I could upgrade the dictionary version to version, but I generally haven't, dictionaries are incredibly large and they're a mind-numbing thing to try to adjust. Version to version I have started suggesting/adjusting which dictionaries I suggest, and with the next version I won't be suggesting the same ones because some of them were sufficiently bad (way too many typos included).